### ISSUE TITLE
DeprecationWarning when importing StyleFrame

### ISSUE THREAD

**Issue by OD1995** [DeepSpace2/StyleFrame#57]
### Issue

When importing StyleFrame I get the following message:
`[.....]\Continuum\anaconda3\lib\site-packages\socks.py:58: DeprecationWarning:Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working`

If nothing else, is there a way to silence this warning?

I have tried the below to no avail.
```
import warnings
warnings.filterwarnings("ignore", category=DeprecationWarning)
```

### Versions

Python : 3.7.0
StyleFrame : 2.0.4
pandas : 0.24.2
openpyxl : 2.5.6

**Comment by DeepSpace2** [comment]
I'm curious as to why this warning comes from `socks.py`, usually it comes from `jsonschema`. Either way this is not caused directly by StyleFrame, but by one of its dependencies (or sub-dependencies) which will have to be updated when Python 3.8 arrives.

Your attempt to silence it did not work because of StyleFrame.deprecations.py line 28:

https://github.com/DeepSpace2/StyleFrame/blob/9721534560009c1ac61c28fcec47c026d6803aca/StyleFrame/deprecations.py#L28

You can:

- Manually remove that line
- Temporarily redirect stderr:

```
    import sys, os
    sys.stderr = open(os.devnull, "w")  # silence stderr
    from StyleFrame import StyleFrame, Styler, utils
    sys.stderr = sys.__stderr__  # unsilence stderr
```
